build/constants.yaml

```diff
@@ -26,14 +26,14 @@
     python-gunicorn-custom-worker-num: PYTHON_GUNICORN_CUSTOM_WORKER_NUM
     python-gunicorn-custom-thread-num: PYTHON_GUNICORN_CUSTOM_THREAD_NUM
   outputs:
-    - type: csharp
```

Comment:
nit: I'm assuming these indentation changes don't affect the resulting code (based on what's in this PR), but do we know why it's occurring?